### ISSUE TITLE
`dnf5 list --upgrades` doesn't respect the options for advisory types/severities and bug/cve/advisory numbers

### ISSUE THREAD

**Issue by pkratoch** [rpm-software-management/dnf5#1816]
In dnf4, `list upgrades` accepts the following options:

--bugfix
--enhancement
--newpackage
--security
--advisory
--bz
--cve
--sec-severity

Dnf5 should filter the upgrades using these options as well, or the differences and how to search for different types of upgrades should be documented.

There are two test scenarios for `list upgrades --security`: https://github.com/rpm-software-management/ci-dnf-stack/blob/main/dnf-behave-tests/dnf/list.feature#L452-L492

**Comment by m-blaha** [comment]
Hm, I'm not sure this is worth implementing. Even in dnf4 this functionality for `list` was not documented (man page says `Applicable for the install,  repoquery,  updateinfo, upgrade and offline-upgrade (dnf-plug‐ins-core) commands.` for all those options.
I think `dnf5 advisory` is better fit for searching different types of upgrades.

**Comment by evan-goode** [comment]
> or the differences and how to search for different types of upgrades should be documented.

Agreed on just documenting this in https://dnf5.readthedocs.io/en/latest/changes_from_dnf4.7.html.
